java-checks/src/main/java/org/sonar/java/symexecengine/DataFlowVisitor.java

```diff
@@ -104,7 +127,11 @@ public void visitTryStatement(TryStatementTree tree) {
     executionState = blockES;
     scan(tree.block());
     scan(tree.resources());
-    handleResources(tree.resources());
+    for (VariableTree resource : tree.resources()) {
+      for (SymbolicValue value : executionState.getValues(resource.symbol())) {
+        check.onTryResourceClosed(executionState, value);
```

Comment:
:new: ![MINOR](https://raw.githubusercontent.com/henryju/image-hosting/master/minor.png)Method 'SymbolicExecutionCheck.onTryResourceClosed(...)' is deprecated.[![...](https://raw.githubusercontent.com/henryju/image-hosting/master/more.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)